repository/p.json

```diff
@@ -1387,6 +1387,17 @@
 				}
 			]
 		},
+		{
+			"name": "Pipenv — Python Dev Workflow for Humans",
```

Comment:
The ` - Python Dev Workflow for Humans` should be in your repo description, not the package name.

Reply:
(It's also using the EM DASH character and not the ASCII hyphen.)